src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -171,6 +171,21 @@ public void TestNormalizeStatement1()
             TestNormalizeStatement("Func<string, int> f = blah;", "Func<string, int> f = blah;");
         }
 
+        [Theory]
+        [InlineData("int*p;", "int* p;")]
+        [InlineData("int *p;", "int* p;")]
+        [InlineData("int*p1,p2;", "int* p1, p2;")]
+        [InlineData("int *p1, p2;", "int* p1, p2;")]
+        [InlineData("int**p;", "int** p;")]
+        [InlineData("int **p;", "int** p;")]
+        [InlineData("int**p1,p2;", "int** p1, p2;")]
+        [InlineData("int **p1, p2;", "int** p1, p2;")]
+        [WorkItem(49733, "https://github.com/dotnet/roslyn/issues/49733")]
+        public void TestNormalizeAsteriskInPointerDeclaration(string text, string expected)
```

Comment:
💡 No need to change this test, but as a side note I've found it beneficial to move `expected` from a parameter to a local variable:

```csharp
string expected = text switch
{
    "int*p;" => "int* p;",
    // ...
    _ => throw ExceptionUtilities.Unreachable,
};
```

This allows the test to better associate with the historical test definition if the expected output ever changes.

Reply:
Don't you have to then duplicate the inputs in the InlineData and the conditions in the switch? I guess I'd absolutely prefer the form in the PR. :-/

Reply:
> Don't you have to then duplicate the inputs in the InlineData and the conditions in the switch?

Not always. This particularly applies if two different tests use the same inputs but have two different expected outputs (e.g. due to a configuration difference under test). Sharing the input definitions (e.g. via `MemberData`) makes it easy to see the tests use the same input set even though there is some duplication of values inside the tests themselves.